### ISSUE TITLE
Security Flaw

### ISSUE THREAD

**Issue by tinnted** [pfn/keepasshttp#362]
Using KeepassHelper extension in opera and firefox and sending "about:home" which is actually mozilla home page url returns all the passwords from database.
All the setting are in default and the worst thing is i dont give permission to any entries, i like to have prompt every time i need a password but it does nothing in this case.

**Comment by pfn** [comment]
This sounds impossible. Unless this is a new feature to render passwords on
the home page

**Comment by tinnted** [comment]
should i attach a screen-shot ? i will also try it in ubuntu and see if the same happens or will try to see if there are any other extensions that use keepasshttp gateway and see if they do the same

**Comment by pfn** [comment]
I suppose, but the likely case is that theres something installed that is
somehow requesting the password list function.

**Comment by m-a-y-k** [comment]
@tinnted I agree, that one must allow each and every access to passwords. From the homepage I understand "user can allow or deny access to single entries" as a promise.
@pfn However, I found a reproducible way to bypass the "Confirm Access" dialog. Please establish a private channel where we can rectify this.
